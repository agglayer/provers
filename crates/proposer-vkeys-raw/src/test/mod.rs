use alloy_primitives::B256;

#[test]
fn aggregation_vkey_hash_snapshot() {
    insta::assert_debug_snapshot!(crate::aggregation::VKEY_HASH);
}

#[test]
fn range_vkey_commitment_snapshot() {
    insta::assert_debug_snapshot!(B256::new(crate::range::VKEY_COMMITMENT));
}
