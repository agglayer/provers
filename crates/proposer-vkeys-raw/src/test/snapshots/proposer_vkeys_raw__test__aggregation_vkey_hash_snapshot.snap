---
source: crates/proposer-vkeys-raw/src/test/mod.rs
assertion_line: 5
expression: "crate::aggregation::VKEY_HASH"
snapshot_kind: text
---
[
    1638338388,
    1256426946,
    526664290,
    1122453194,
    1403846808,
    601906877,
    1176579776,
    312933952,
]
